apps/full-site-editing/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+	extends: require.resolve( '@automattic/calypso-build/babel.config.js' ),
+	presets: [ require( '@wordpress/babel-preset-default' ) ],
```

Comment:
You might want to consider including only relevant bits shipped in Calypso-build for this purpose:

```js
presets: [ require( '@automattic/calypso-build/babel/wordpress-element' ) ],
```

The [`@wordpress/default`](https://github.com/WordPress/gutenberg/blob/ddf0cd701d554bd6ec6a4db470aa24b7abbb703f/packages/babel-preset-default/index.js) is little more extensive.